src/open_producten/utils/filters.py

```diff
@@ -66,3 +105,120 @@ def filter(self, qs, value):
 
         qs = self.get_method(qs)(**{lookup: value, language_lookup: language_code})
         return qs
+
+
+class Operators(models.TextChoices):
+    exact = "exact", _("gelijk aan")
+    gt = "gt", _("grooter dan")
+    gte = "gte", _("grooter dan of gelijk aan")
+    lt = "lt", _("kleiner dan")
+    lte = "lte", _("kleiner dan of gelijk aan")
+    icontains = "icontains", _("case-insensitive partial match")
```

Comment:
Since the others are also in Dutch
```suggestion
    icontains = "icontains", _("hoofdletterongevoelige gedeeltelijke match")
```